### PR TITLE
Contributing revamp, take 2

### PR DESCRIPTION
**Pull request by OdysLam**
The final piece of a series of PRs across our repositories to align them in terms of contribution and community.

In short:

Remove the community health files (code of conduct, contributing, etc.) from every repo that they exist. They will be automatically sourced from the .github repository (GH docs).
Create new default community health files
Point Learn to source these files from .github
Revamp contributing guide and contributor journey
Align the README of main community-facing repos (netdata/netdata, netdata/dashboard, netdata/helmchart, netdata/go.d, netdata/community) as far as contributing and community guidelines are concerned.
Feedback is much appreciated :)

P.S contributors is removed because it is no longer updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,14 @@ Netdata](https://user-images.githubusercontent.com/1153921/96495792-2e881380-11f
 
 ## Menu
 
+- [Menu](#menu)
```
Comment (joelhans) [inline README.md:45]: I don't think we need this very self-referential item.

File: README.md [hunk 2]
```diff
@@ -190,11 +195,14 @@ to collect metrics, troubleshoot via charts, export to external databases, and m
 
 ## Community
 
-You can find most of the Netdata team in our [community forum](https://community.netdata.cloud). It's the best place to
-ask questions, find resources, and get to know the Netdata team.
+Netdata is striving to be an all inclusive open-source project and community. Please read our [Code of Conduct](https://learn.netdata.cloud/contribute/code-of-conduct).
```
Comment (joelhans) [inline README.md:198]: Let's be a bit more confident here.

```suggestion
Netdata is an inclusive open-source project and community. Please read our [Code of Conduct](https://learn.netdata.cloud/contribute/code-of-conduct).
```

File: README.md [hunk 3]
```diff
@@ -190,11 +195,14 @@ to collect metrics, troubleshoot via charts, export to external databases, and m
...
+
+You can find most of the Netdata team in our [community forums](https://community.netdata.cloud). It's the best place to
```
Comment (joelhans) [inline README.md:200]:
```suggestion
Find most of the Netdata team in our [community forums](https://community.netdata.cloud). It's the best place to
```

File: README.md [hunk 4]
```diff
@@ -203,10 +211,12 @@ You can also find Netdata on:
 
 ## Contribute
 
-We welcome [contributions](/CONTRIBUTING.md) to our code and to our
-[documentation](/docs/contributing/contributing-documentation.md). Feel free to join the team!
+Contributions are the lifeblood of OSS projects, such as Netdata Agent. While we are continuing to invest resources and improve the OSS Agent, we need every help we can get in our effort to democratize monitoring!
```
Comment (joelhans) [inline README.md:214]:
```suggestion
Contributions are the lifeblood of open-source projects. While we continue to invest in and improve Netdata, we need help to democratize monitoring!
```

File: README.md [hunk 5]
```diff
@@ -203,10 +211,12 @@ You can also find Netdata on:
...
 
-To report bugs or get help, use [GitHub's issues](https://github.com/netdata/netdata/issues).
+- Read our [Contributing Guide](https://learn.netdata.cloud/contribute/handbook). In there, you will find all the information you need to contribute to the Netdata Agent, from improving our documentation to engaging in the community and development. We have made efforts to make it as frictionless as possible, but if you need help, just ping us on our community forums!
+- We have a whole category dedicated to contributing and extending the Netdata Agent on our [community forums](https://community.netdata.cloud/c/agent-development/9)
+- You have found a bug? Open a [GitHub issue](https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md).
+- View our [Security Policy](https://github.com/netdata/netdata/security/policy)
```
Comment (joelhans) [inline README.md:219]:
```suggestion
- Read our [Contributing Guide](https://learn.netdata.cloud/contribute/handbook), which contains all the information you need to contribute to Netdata, such as improving our documentation, engaging in the community, and developing new features. We've made it as frictionless as possible, but if you need help, just ping us on our community forums!
- We have a whole category dedicated to contributing and extending Netdata on our [community forums](https://community.netdata.cloud/c/agent-development/9)
- Found a bug? Open a [GitHub issue](https://github.com/netdata/netdata/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md).
- View our [Security Policy](https://github.com/netdata/netdata/security/policy).
```

**Review by joelhans — CHANGES_REQUESTED** [review]
Does sourcing from the new `.github` repo happen automatically, or is that some manual function you'll do once this PR is merged and the files deleted?

**Comment by ilyam8** [comment]
i wonder why netdatabot is a reviewer 🤖 😄

**Comment by joelhans** [comment]
@OdysLam This LGTM, but you've got some issues with the changelog. I would recommend just aligning it to the latest version so that it doesn't keep bothering you about whatever you might have changed there.

**Review by ilyam8 — APPROVED** [review]

**Review by joelhans — APPROVED** [review]

**Pull request merged**